decidim-proposals/app/controllers/decidim/proposals/admin/proposals_controller.rb

```diff
@@ -6,7 +6,9 @@ module Admin
       # This controller allows admins to manage proposals in a participatory process.
       class ProposalsController < Admin::ApplicationController
         helper Proposals::ApplicationHelper
-        helper_method :proposals
+        helper_method :proposals, :query
+
+        def index; end
```

Comment:
Rails doesn't need the action definition is there's already the view file, so we can delete this!